src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,51 @@
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** This servlet class is responsible the profile pages*/
+public class ProfileServlet extends HttpServlet{
+	
+	  /** From my perspective this variable is used to keep track of who is looking at
```

Comment:
The indentation is a little off from the one in the standard -- instead of tabs, there should be two spaces.

Reply:
Ditto on this. I'll try to get us set up with a linter soon that will catch formatting issues automatically. :smile:

Reply:
https://www.10bestdesign.com/dirtymarkup/js/

This websites has some cool formatting features for a couple languages